taca/illumina/HiSeq_Runs.py

```diff
@@ -661,13 +661,14 @@ def _generate_clean_samplesheet(self, ssparser):
                     entry[_data_filed_conversion(field)] = value
             data.append(entry)
 
-        fields_to_output = ['Lane', 'Sample_ID', 'Sample_Name', 'index', 'index2', 'Sample_Project']
+        fields_to_output = ['Lane', ssparser.dfield_sid, ssparser.dfield_snm, 'index', 'index2', ssparser.dfield_proj]
         #now create the new SampleSheet data section
         output+="[Data]{}".format(os.linesep)
         for field in ssparser.datafields:
-            new_field = _data_filed_conversion(field)
-            if new_field not in fields_to_output:
-                fields_to_output.append(new_field)
+            if field not in fields_to_output:
```

Comment:
ok... this starts to be complicated :-D 
We really need to change how we generate sample sheet, this solution is fine for now